go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/spf13/cast v1.4.1
 	github.com/spf13/cobra v1.4.0
 	github.com/stretchr/testify v1.7.1
-	github.com/tendermint/fundraising v0.3.0
+	github.com/tendermint/fundraising v0.3.1-0.20220613014523-03b4a2d4481a
```

Comment:
can we avoid using the commit version and use only `v0.3.1`?

Reply:
We need a recent release of `fundraising` that includes after auction creation hooks

For final version, we will use `fundraising` post audit merge